feat: otter training sdk - welcome and other small fixes (#2524)
AmadeusITGroup/otter: apps/showcase/src/assets/trainings/sdk/steps/welcome/instructions.md

This interactive tutorial will teach you how to generate and use an SDK using the Otter framework.

### How to use this tutorial
This tutorial is composed of several steps, including:
1. How to use the Otter SDK?
2. Customize your fetch client with plugins
3. Integrate your SDK in Angular
4. Generate your first SDK - Specifications
5. Generate your first SDK - Command
6. SDK with Dates - How to use
7. SDK with Dates - Generation
8. SDK with model extension

We will cover explanations on these topics along with examples and exercises for better understanding.

### Instructions for exercises

> As mentioned, each step covers a concept of the Otter SDK. You can do one step or all of them.

Reviewer: ```suggestion
Each step covers a concept of the Otter SDK and can be done independently. 
```

To navigate between the steps, you can use the arrows or the drop-down menu at the top right of the page.

Most of the exercises can be done in the integrated code editor (except for the generation of an SDK, which should be done locally).
Each step's objective and exercise will be described in the instructions, and you can check out your results in the **Preview** tab. 
You can also check out the **Output** tab for the console. You can interact with the console and run commands in the **Terminal** tab.

When a solution is available, you can reveal the answer by clicking on the **Show Solution** button at the top right of the page.

### Prerequisites

> You will need some familiarity with HTML and Typescript to understand the exercises.

Reviewer 2: Maybe some knowledge on OpenAPI as mentioned later

> You will need some familiarity with HTML and Typescript to understand the exercises.

Reviewer: it's only for the step with `Angular` in the name I think, I wouldn't mention it in the global prerequisites

> You will need some familiarity with HTML and Typescript to understand the exercises.

Author: @matthieu-crouzet what do you think?

#### Environment
For the exercises that need to be done locally, your environment requires:
- <a href="https://nodejs.org/en/learn/getting-started/how-to-install-nodejs" target="_blank">Node</a> - Version 18.0.0 or above - minimum required version according to the <a href="https://github.com/AmadeusITGroup/otter/blob/main/packages/%40ama-sdk/schematics/package.json" target="_blank">&#64;ama-sdk/schematics package</a>
- <a href="https://adoptium.net/" target="_blank">Java</a> _(for SDK generation)_ - Version 11 or above - minimum required version according to <a href="https://github.com/OpenAPITools/openapi-generator" target="_blank">OpenAPI Generator</a>

#### Resources

> We would like to specify that this is not an OpenAPI training. We invite you to familiarize yourself with OpenAPI and how to configure the generation SDK prior to starting this tutorial.
> We will refer to some OpenAPI concepts throughout this training when explaining our configuration options built on top of OpenAPI.
> 
> - <a href="https://swagger.io/docs/specification/about/" target="_blank">What is OpenAPI?</a>
> - <a href="https://swagger.io/specification/" target="_blank">OpenAPI Specification</a>

Reviewer: ```suggestion
We are trying to make our training as accessible as possible, but for the best experience we recommend getting familiar with OpenAPI before starting the exercises.
We will refer to some OpenAPI concepts throughout this training when explaining our configuration options built on top of OpenAPI.
Here are some useful links to get started:
- <a href="https://swagger.io/docs/specification/about/" target="_blank">What is OpenAPI?</a>
- <a href="https://swagger.io/specification/" target="_blank">OpenAPI Specification</a>
```

Reviewer 2: I would move the text part inside the `prerequisites` section and keep the links here

Author: This is already a subsection of `Prerequisites`, should I move it above to make it more clear?

Reviewer 2: I would say yes to have OpenAPI referenced at the same level as HTML and TypeScript
I would also have the resources section outside the prerequisites because we may add some resources not linked to the prerequisite and people don't have to read all resources before doing the training so it's not really a prerequisite
I don't know what you think

Author: Good idea, I will move the first line to the prerequisites and keep the rest under the section `Resources` which will be its own section
